Part of https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/3121

In file included from layers/synchronization2.cpp:33:
utils/generated/vk_safe_struct.h:3503:5: error: unknown type name 'VkVideoCapabilitiesFlagsKHR'; did you mean 'VkVideoCapabilityFlagsKHR'?
    VkVideoCapabilitiesFlagsKHR capabilityFlags;
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~
    VkVideoCapabilityFlagsKHR
/usr/local/include/vulkan/vulkan_beta.h:74:17: note: 'VkVideoCapabilityFlagsKHR' declared here
typedef VkFlags VkVideoCapabilityFlagsKHR;
                ^
In file included from layers/synchronization2.cpp:33:
utils/generated/vk_safe_struct.h:5546:5: error: unknown type name 'VkVideoEncodeH264CapabilitiesFlagsEXT'; did you mean 'VkVideoEncodeH264CapabilityFlagsEXT'?
    VkVideoEncodeH264CapabilitiesFlagsEXT flags;
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    VkVideoEncodeH264CapabilityFlagsEXT
/usr/local/include/vulkan/vulkan_beta.h:453:17: note: 'VkVideoEncodeH264CapabilityFlagsEXT' declared here
typedef VkFlags VkVideoEncodeH264CapabilityFlagsEXT;
                ^
In file included from layers/synchronization2.cpp:33:
utils/generated/vk_safe_struct.h:5735:5: error: unknown type name 'VkVideoDecodeH264FieldLayoutFlagsEXT'; did you mean 'VkVideoDecodeH264PictureLayoutFlagsEXT'?
    VkVideoDecodeH264FieldLayoutFlagsEXT fieldLayout;
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    VkVideoDecodeH264PictureLayoutFlagsEXT
/usr/local/include/vulkan/vulkan_beta.h:579:17: note: 'VkVideoDecodeH264PictureLayoutFlagsEXT' declared here
typedef VkFlags VkVideoDecodeH264PictureLayoutFlagsEXT;
                ^

--- utils/generated/vk_safe_struct.cpp.orig	2021-07-21 20:54:27 UTC
+++ utils/generated/vk_safe_struct.cpp
@@ -25624,7 +25624,7 @@ void safe_VkVideoEncodeH264ProfileEXT::initialize(cons
 safe_VkVideoDecodeH264ProfileEXT::safe_VkVideoDecodeH264ProfileEXT(const VkVideoDecodeH264ProfileEXT* in_struct) :
     sType(in_struct->sType),
     stdProfileIdc(in_struct->stdProfileIdc),
-    fieldLayout(in_struct->fieldLayout)
+    pictureLayout(in_struct->pictureLayout)
 {
     pNext = SafePnextCopy(in_struct->pNext);
 }
@@ -25638,7 +25638,7 @@ safe_VkVideoDecodeH264ProfileEXT::safe_VkVideoDecodeH2
 {
     sType = copy_src.sType;
     stdProfileIdc = copy_src.stdProfileIdc;
-    fieldLayout = copy_src.fieldLayout;
+    pictureLayout = copy_src.pictureLayout;
     pNext = SafePnextCopy(copy_src.pNext);
 }
 
@@ -25651,7 +25651,7 @@ safe_VkVideoDecodeH264ProfileEXT& safe_VkVideoDecodeH2
 
     sType = copy_src.sType;
     stdProfileIdc = copy_src.stdProfileIdc;
-    fieldLayout = copy_src.fieldLayout;
+    pictureLayout = copy_src.pictureLayout;
     pNext = SafePnextCopy(copy_src.pNext);
 
     return *this;
@@ -25667,7 +25667,7 @@ void safe_VkVideoDecodeH264ProfileEXT::initialize(cons
 {
     sType = in_struct->sType;
     stdProfileIdc = in_struct->stdProfileIdc;
-    fieldLayout = in_struct->fieldLayout;
+    pictureLayout = in_struct->pictureLayout;
     pNext = SafePnextCopy(in_struct->pNext);
 }
 
@@ -25675,7 +25675,7 @@ void safe_VkVideoDecodeH264ProfileEXT::initialize(cons
 {
     sType = copy_src->sType;
     stdProfileIdc = copy_src->stdProfileIdc;
-    fieldLayout = copy_src->fieldLayout;
+    pictureLayout = copy_src->pictureLayout;
     pNext = SafePnextCopy(copy_src->pNext);
 }
 #endif // VK_ENABLE_BETA_EXTENSIONS
--- utils/generated/vk_safe_struct.h.orig	2021-07-21 20:54:27 UTC
+++ utils/generated/vk_safe_struct.h
@@ -3500,7 +3500,7 @@ struct safe_VkVideoProfilesKHR {
 struct safe_VkVideoCapabilitiesKHR {
     VkStructureType sType;
     void* pNext;
-    VkVideoCapabilitiesFlagsKHR capabilityFlags;
+    VkVideoCapabilityFlagsKHR capabilityFlags;
     VkDeviceSize minBitstreamBufferOffsetAlignment;
     VkDeviceSize minBitstreamBufferSizeAlignment;
     VkExtent2D videoPictureExtentGranularity;
@@ -5543,7 +5543,7 @@ struct safe_VkImageViewAddressPropertiesNVX {
 struct safe_VkVideoEncodeH264CapabilitiesEXT {
     VkStructureType sType;
     const void* pNext;
-    VkVideoEncodeH264CapabilitiesFlagsEXT flags;
+    VkVideoEncodeH264CapabilityFlagsEXT flags;
     VkVideoEncodeH264InputModeFlagsEXT inputModeFlags;
     VkVideoEncodeH264OutputModeFlagsEXT outputModeFlags;
     VkExtent2D minPictureSizeInMbs;
@@ -5732,7 +5732,7 @@ struct safe_VkVideoDecodeH264ProfileEXT {
     VkStructureType sType;
     const void* pNext;
     StdVideoH264ProfileIdc stdProfileIdc;
-    VkVideoDecodeH264FieldLayoutFlagsEXT fieldLayout;
+    VkVideoDecodeH264PictureLayoutFlagsEXT pictureLayout;
     safe_VkVideoDecodeH264ProfileEXT(const VkVideoDecodeH264ProfileEXT* in_struct);
     safe_VkVideoDecodeH264ProfileEXT(const safe_VkVideoDecodeH264ProfileEXT& copy_src);
     safe_VkVideoDecodeH264ProfileEXT& operator=(const safe_VkVideoDecodeH264ProfileEXT& copy_src);
